src/main/resources/templates/layout/header.ftl

```diff
@@ -11,11 +11,12 @@
             <div class="header-search-select-item">
                 <select data-placeholder="All Categories" class="chosen-select">
                     <option>All Categories</option>
-                    <option>Shops</option>
-                    <option>Hotels</option>
-                    <option>Restaurants</option>
-                    <option>Fitness</option>
-                    <option>Events</option>
+                    <option>Sub-lease</option>
+                    <option>temp</option>
+                    <option>long-tern</option>
+                    <option>houses</option>
+                    <option>apartment</option>
+                    <option>more...</option>
                 </select>
             </div>
             <button class="header-search-button" onclick="window.location.href='listing.html'">Search</button>
```

Comment:
how to pass the selector value to controller?

Reply:
You can take contact page as a sample:
* use `name` attribute for `input` or `select`
https://github.com/findeasily/findeasily.net/blob/master/src/main/resources/templates/public/contact.ftl#L48
* get its value with `@RequestParam `
https://github.com/findeasily/findeasily.net/blob/master/src/main/java/net/findeasily/website/controller/ajax/PublicAjaxController.java#L149